app/src/main/java/at/bitfire/icsdroid/model/CreateSubscriptionModel.kt

```diff
@@ -20,10 +20,10 @@ class CreateSubscriptionModel(application: Application) : AndroidViewModel(appli
     private val subscriptionsDao = database.subscriptionsDao()
     private val credentialsDao = database.credentialsDao()
 
-    val success = MutableLiveData(false)
-    val errorMessage = MutableLiveData<String?>(null)
-    val isCreating = MutableLiveData(false)
-    val showNextButton = MutableLiveData(false)
+    val success = mutableStateOf(false)
+    val errorMessage = mutableStateOf<String?>(null)
+    val isCreating = mutableStateOf(false)
+    val showNextButton = mutableStateOf(false)
```

Comment:
As discussed in the internal meeting, we agreed on using `var`+`by`+`private set`, so this would be:
```kotlin
var success by mutableStateOf(false)
  private set
var errorMessage by mutableStateOf<String?>(null)
  private set
var isCreating by mutableStateOf(false)
  private set
var showNextButton by mutableStateOf(false)
  private set
```
That way no `.value` calls are required, and in Composables just use `=`.

Reply:
I will use the new UiState data class :+1: